### DISCUSSION TITLE
New to DaVinci Resolve?

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#400] [Website Discussions]
# New to DaVinci Resolve?

If you're brand new to DaVinci Resolve- this is the place for you!

https://resolve.cafe/newbies/

**Comment by Van-nextpay** [comment]
모델의 워킹을 만들어 줘

**Comment by Van-nextpay** [comment]
모델 워킹을 10초 동영상으로 만들어 줘
